File: Player.py
import random

class Player:
    def __init__(self, name, pos=0):
        self.name = name
        self.pos = pos
    
    def getName(self):
        return self.name
    
    def getPos(self):
        return self.pos
    
    def move(self):
        self.pos += random.randint(0,5)

File: Player.go
package Player

import (
	"math/rand"
	"time"
)

type Player struct {
	Name string
	Pos  int
}

func (p *Player) GetName() string {
	return p.Name
}

func (p *Player) GetPost() int {
	return p.Pos
}

func (p *Player) Move() {
	rand.Seed(time.Now().UnixNano())
	p.Pos += rand.Intn(5)
}
